#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("power_per_pixel_threshold", double_t, 0, "Action detection threshold", 0.05,  0, 1)

exit(gen.generate("decopin_hand", "decopin_hand", "DecopinHand"))
